src/nupic/regions/TPRegion.py

```diff
@@ -42,11 +42,13 @@ def _getTPClass(temporalImp):
     return TP10X2.TP10X2
   elif temporalImp == 'tm_py':
     return TP_shim.TPShim
+  elif temporalImp == 'fast_tm_py':
```

Comment:
But then it would be inconsistent with the `TemporalMemory` and `FastTemporalMemory` naming.